app/models/like.rb

```diff
@@ -1,4 +1,11 @@
 class Like < ApplicationRecord
   belongs_to :user
   belongs_to :post
+
+  after_create :update_likes_counter
+  after_destroy :update_likes_counter
+
+  def update_likes_counter
+    post.update(likes_counter: post.likes.count)
+  end
 end
```

Comment:
[Suggestion]: It would be nice to use one of the [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) - after_save to call the #update_likes_counter method every time a new post has been saved to database, then make the update_likes_counter method private.